projects/plugins/jetpack/modules/google-fonts/load.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Load the google fonts based on the current WordPress version.
+ *
+ * @package automattic/jetpack
+ */
+
+if ( class_exists( 'WP_Font_Library' ) && class_exists( 'WP_Font_Face' ) && class_exists( 'WP_Font_Collection' ) ) {
```

Comment:
This won't be available in WordPress 6.4 since it's currently only available in Gutenberg.  It's planned to be available in WordPress 6.5.  However this technique IS available in situations other than WordPress 6.4 (or 6.5).

Reply:
Are there any changes I have to make here? I made a change to only check the `WP_Font_Face` class as I assume we still rely on it to generate and print font styles. WDYT?

Reply:
The class isn't actually used or depended on at all as the logic exists now, WP_Font_Collection WAS used but now that collection data is just loaded manually and the class isn't needed.  The data provided in that file that is loaded is everything needed for the fonts to work; no extra logic needed. That allows the logic to be font library/api agnostic.  Doing it like this means that it ONLY works in Gutenberg, wereas it COULD work WITHOUT Gutenberg.  Considering that it HASN'T worked without Gutenberg in the past I don't know if that matters though.

So I guess the answer is "If you want this to work WITHOUT gutenberg it CAN you'll just need to change that if() logic a bit."  "If you don't care for this to work without Gutenberg then it's fine and dandy as it is"

Reply:
Okay. I'll leave it as it is since it hasn't worked without Gutenberg in the past. I'd prefer to make it work without Gutenberg after WordPress 6.5 or above which supports the `WP_Font_Face` class.